internal/kafka/internal/cmd/supportedkafkainstancetypes/flags.go

```diff
@@ -0,0 +1,8 @@
+package supportedkafkainstancetypes
+
+const (
+	// FlagCloudProvider is a flag representing cloud provider Id
+	FlagCloudProvider = "cloud-provider"
```

Comment:
should this be cloud-provder-id  to add more clarity?

Reply:
I agree. I will be renaming this

Reply:
I added a new commit with the change

Reply:
thanks Migurel